web/src/javascript/game.js

```diff
@@ -6,10 +6,33 @@ import {
   initMap,
 } from "./drawMap";
 
+const apiURL = "http://localhost:1337/api/1/game/";
+
+function extractPointsFromState(gameState) {
```

Comment:
`extractAllCellsFromState`? My understanding is that this returning the cells not just their positions, right?